### ISSUE TITLE
Missing "lang" attribute from <html> tag

### ISSUE THREAD

**Issue by jimafisk** [CityOfBoston/boston.gov-d7#1278]
#### Expected Behavior/Acceptance Criteria
The `<html>` tag should have a "lang" attribute for [accessibility reasons](https://developer.paciellogroup.com/blog/2016/06/using-the-html-lang-attribute/).
 
Here is an example of this working properly:
![lang_working](https://user-images.githubusercontent.com/5913244/53895461-18b1e480-4000-11e9-97ab-a77359179960.png)

#### Actual Behavior
The current live site does not have this attribute:
![missing_lang](https://user-images.githubusercontent.com/5913244/53895344-d7b9d000-3fff-11e9-9982-f5f071b46671.png)

We tried to fix this previously via https://github.com/CityOfBoston/boston.gov-d7/pull/1248, which appears to work locally (first screenshot) but does not work on the live server (second screenshot).

#### Browser
Tested in Firefox and Chrome

#### Steps to Reproduce the Behavior/How to Test
Go to the homepage and inspect the element.

**Comment by jimafisk** [comment]
I think I actually know what happened here...

The lang update was grouped with this related_content release: https://github.com/CityOfBoston/boston.gov-d7/pull/1250

That release was later reverted here: https://github.com/CityOfBoston/boston.gov-d7/pull/1262

**Comment by mmcgowanBOS** [comment]
It looks like the change was in fact in the develop branch already, but for some reason wasn't included int he last merge with master. I forced a new PR and this seems to fix the problem. See PR below for reference.
https://github.com/CityOfBoston/boston.gov-d7/pull/1279
